pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentDispatcherMultipleConsumers.java

```diff
@@ -862,11 +864,15 @@ public void cursorIsReset() {
         }
     }
 
-    protected boolean addMessageToReplay(long ledgerId, long entryId) {
+    protected boolean addMessageToReplay(long ledgerId, long entryId, Long stickyKeyHash) {
```

Comment:
Would it be possible to omit the usage of java.lang.Long object ? the boxing from long -> Long will create object instances and doesn't fit well in the goal of minimizing allocations in the Pulsar code base.

Reply:
@lhotari do you have a proposal ? we cannot use a sentinel value, because every "long" value is allowed here

Reply:
@eolivelli yes, that's true. I think it's better that the PR author solves the issue in the way that makes sense than proposing an exact change to do. In my review comment I'm just bringing it to attention that auto-boxing long -> Long will create a java.lang.Long object instance. That's something that isn't desired in the Pulsar code base for hot code paths since there's a goal of minimizing allocations.

Reply:
Fixed.